packages/useScript/src/useScript.ts

```diff
@@ -33,19 +69,27 @@ export default function useScript(url: string): boolean {
       }
     } else {
       if (script.getAttribute('data-loaded') === 'true') {
-        setReady(true)
         // Already loaded, so we can return early
-        return () => {}
+        return onReady()
+      }
+
+      if (script.getAttribute('data-failed') === 'true') {
```

Comment:
It might make sense to use `data-loaded="false"` instead of `data-failed` - I know i added the `data-failed` originally, but I think it makes more sense to just use one attribute. :)